cider-completion.el

```
@@ -116,14 +116,16 @@ form, with symbol at point replaced by __prefix__."
             (scan-error nil)
             (user-error nil)))
     (save-excursion
-      (let* ((pref-end (point))
-             (pref-start (cider-completion-symbol-start-pos))
-             (context (cider-defun-at-point))
-             (_ (beginning-of-defun))
-             (expr-start (point)))
-        (concat (when pref-start (substring context 0 (- pref-start expr-start)))
-                "__prefix__"
-                (substring context (- pref-end expr-start)))))))
+      (condition-case _
```

This can be replaced by `ignore-errors`. I'd also add some explanations why we're doing in the code, as suppressing errors is rarely a good idea.